docs/faq.md

```diff
@@ -99,17 +99,35 @@ We try to use default descriptors and derivation paths exactly for this reason -
 
 WIP means that we don't try to be very backward-compatible at the moment. At some point we may change wallet storage format for example, and you would need to migrate using some script or create wallets from scratch. In this case, we would provide migration scripts.
 
-## *What's the difference between specter-desktop and specter-DIY?*
+## *What's the difference between specter-desktop and specter-DIY?* 
 
 Specter-desktop is a watch-only GUI software wallet running on Bitcoin Core using its wallet and full node functionality.
 Bitcoin Core tracks addresses, UTXO (unspent transaction outputs) and composes PSBT (partially-signed bitcoin transactions).
 
 Whereas, [Specter-DIY](https://github.com/cryptoadvance/specter-diy) is a do-it-yourself hardware wallet from off the shelf components, that signs and broadcasts transactions using QR codes that forgets your private keys when powered off.
 
+## *Is a full node necessary for using specter-desktop?*
+
+Yes, a bitcoin full node is needed to provide the Bitcoin Core data so you don't rely on 3rd parties, and also for its watch-only wallet capablities.
+
+## *Can I use pruned mode?*
+
+Yes, but if you have many older addresses you will need to re-download the blockchain in order to see your balance and transaction history, which will take some time. Alternatively, you can re-download the blockchain with `blockfilters=1` added to the `bitcoin-cli` rescan command to speed up the process.
```

Comment:
> Alternatively, you can re-download the blockchain with `blockfilters=1` added to the `bitcoin-cli` rescan command to speed up the process.

This is not true for pruned nodes, blockfilters are not supported by pruned nodes.